Source/ARTAuth.m

```diff
@@ -275,13 +275,13 @@ - (void)executeTokenRequest:(ARTTokenRequest *)tokenRequest callback:(void (^)(A
 
 - (void)authorise:(ARTTokenParams *)tokenParams options:(ARTAuthOptions *)authOptions callback:(void (^)(ARTTokenDetails *, NSError *))callback {
     BOOL requestNewToken = NO;
-    ARTAuthOptions *mergedOptions = [self mergeOptions:authOptions];
-    [self storeOptions:mergedOptions];
-    ARTTokenParams *currentTokenParams = [self mergeParams:tokenParams];
+    ARTAuthOptions *currentAuthOptions = authOptions ? authOptions : self.options;
+    [self storeOptions:currentAuthOptions];
+    ARTTokenParams *currentTokenParams = tokenParams ? tokenParams : _tokenParams;
     [self storeParams:currentTokenParams];
 
     // Reuse or not reuse the current token
-    if (mergedOptions.force == NO && self.tokenDetails) {
+    if (currentAuthOptions.force == NO && self.tokenDetails) {
```

Comment:
If I remove `force` property from `AuthOptions` object - it will break `RSA10d` & also `RTC8` requirements, because both them needs `force`.
That’s why in my opinion it's impossible to combine both `RSA10j` and `RSA10d` changes in one PR.